app/src/main/java/com/gathering/android/event/myevent/addevent/invitation/AttendeeListAdapter.kt

```diff
@@ -0,0 +1,70 @@
+package com.gathering.android.event.myevent.addevent.invitation
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.gathering.android.common.ImageLoader
+import com.gathering.android.databinding.ItemAttendeeBinding
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+class AttendeeListAdapter @Inject constructor(
+    @ApplicationContext private val context: Context,
```

Comment:
why do we need to inject "@ApplicationContext" if we already have an inject constructor?

Reply:
As far as I know, when we create an instant of one class in DI and pass the required context in this case Application context we can avoid putting application context annotation here but in our case, we don't provide instance of  this adapter in our DI.